cmd/hyperkube/hyperkube.go

```diff
@@ -154,7 +170,23 @@ func (hk *HyperKube) Run(args []string, stopCh <-chan struct{}) error {
 		}
 	}
 
-	s, err := hk.FindServer(serverName)
+	var s *Server
+	var err error
+	if serverName == "alpha" {
+		if len(args) > 0 && len(args[0]) > 0 {
+			serverName = args[0]
+			args = args[1:]
+			hk.Printf("Warning: alpha command syntax is unable!\n\n")
```

Comment:
is unable to _what_?

Reply:
Took me a second, it's supposed to be an instability warning. `s/unsable/unstable/g`. Will push a new commit and can squash at the end.